src/lib/fb.js

```diff
@@ -61,26 +79,62 @@ export function userRef() {
   return firestore.collection("users").doc(state.user.uid);
 }
 
-export function updateUrl(url) {
+/**
+ * Update's the user's row in Firestore (if signed in) with an updated URL and optional audit time.
+ *
+ * @param {string} url to update the user's row with
+ * @param {!Date} auditedOn of the most recent Lighthouse run
+ * @return {!Date} the earliest audit seen for this URL
+ */
+export async function updateUrl(url, auditedOn = null) {
```

Comment:
I was trying to think of a more specific name for this function since "update" could mean a number of things.
Some options... `setCurrentUrl`, `replaceCurrentUrl`, `addOrReplaceCurrentUrl`?

Reply:
`saveUserUrl`? It's about saving it to Firestore.